modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -9,25 +9,6 @@ This information is necessary to connect to the servers, run the discovery proto
 
 Neo4j provides several mechanisms for cluster members to discover each other and form a cluster based on the configuration and the environment in which the cluster is running, as well as the version of Neo4j being used.
 
```

Comment:
```suggestion

[IMPORTANT]
====
In Neo4j 5.23, the discovery service v2 has been introduced.
It is very crucially important to switch to the the new version, v2, before you start using Neo4j 2025.01.
====

```

Reply:
I'd still be inclined to remove the notion of v2 here